exporters/otlp/src/otlp_grpc_client.cc

```diff
@@ -78,16 +78,20 @@ class OPENTELEMETRY_LOCAL_SYMBOL OtlpGrpcAsyncCallData : public OtlpGrpcAsyncCal
   virtual ~OtlpGrpcAsyncCallData() {}
 };
 }  // namespace
+#endif
 
 struct OtlpGrpcClientAsyncData
 {
+
   std::chrono::system_clock::duration export_timeout = std::chrono::seconds{10};
 
   // The best performance trade-off of gRPC is having numcpu's threads and one completion queue
   // per thread, but this exporter should not cost a lot resource and we don't want to create
-  // too many threads in the process. So we use one completion queue.
-  grpc::CompletionQueue cq;
-
+  // too many threads in the process. So we use one completion queue and shared context.
```

Comment:
So use one completion queue and shared context reduces resource requirement, but also cloud lead to decreased performance?

Reply:
I use gRPC's async APIs, which utilize background threads for notifications of completions. By reusing the internal `CompletionQueue`, there's no need to create an additional background thread for `OtlpGrpcClient`.